numpy/polynomial/_polybase.py

```diff
@@ -682,6 +682,19 @@ def degree(self):
         degree : int
             Degree of the series, one less than the number of coefficients.
 
+        Examples
+        --------
+
+        Create a polynomial object for ``1 + 7*x + 4*x**2``:
+        >>> import numpy as np
+        >>> poly = np.polynomial.Polynomial([1, 7, 4])
+        >>> poly
+        Polynomial([1., 7., 4.], domain=[-1,  1], window=[-1,  1])
+
+        Calculate the degree:
+        >>> poly.degree()
+        2
+
```

Comment:
Just a couple thoughts here:
 - the polynomial classes actually support a "pretty" `__str__` that uses unicode by default
 - The `import numpy as np` is not necessary
 - There must be blank lines between narrative text and executable examples in the `Examples` section, otherwise the doctest parsing will fail (the fact that this *isn't* happening in CI indicates that there are likely problems with our CI coverage!)

Taken together, I'd suggestion something like:

```suggestion
        >>> poly = np.polynomial.Polynomial([1, 7, 4])
        >>> print(poly)
        1.0 + 7.0·x + 4.0·x²
        >>> poly.degree()
        2

```

Feel free to adopt/ignore suggestions as you see fit!